docs/scopes.md

```diff
@@ -0,0 +1,32 @@
+# Scopes
+
+Because not all API clients need access to every publiq API, we work with a permission model to limit your client's access to the APIs that you need.
+
+This permission works through *scopes* that are granted to your API client on publiq's authorization server. Each API requires 1 specific scope. For example to access the UiTPAS API, your client will need to have the `https://api.publiq.be/auth/uitpas` scope.
+
+When requesting a [client access token](./client-access-token.md) or [user access token](./user-access-token.md), you also need to explicitly list which scopes should be included in the token. Tokens that do not include a specific API's scope cannot be used on that API. Note that you cannot request to include scopes that your client is not allowed to have access to.
+
+When using [client identification](./client-identification.md), you do not need to request scopes because you do not need to request a token. Instead the API will look up your client's allowed scopes on the authorization server using the client id included in your request.
```

Comment:
... you may omit the scope(s) in your request. The authorization server will return a token including all the scopes your client is entitled to use.

Reply:
Ik denk dat je suggestie gaat over client _access tokens_, maar het gaat hier over client _identification_. Als dat niet duidelijk was voor jou al, dan moeten we het misschien wel duidelijker proberen maken voor andere lezers.

Reply:
Zie 120ebdf